src/main/java/net/floodlightcontroller/websocket/WebSocketManager.java

```diff
@@ -0,0 +1,257 @@
+		
+		instance = this; //singleton to assure one is being called
+		if(VERBOSE){logger.info("made it before startup");}
+		
+		Server server = new Server();
+		ServerConnector connector = new ServerConnector(server);
+		connector.setPort(8111);
+		server.addConnector(connector);
+	     
+
+		// Setup server endpoint for this application at "/"
+		ServletContextHandler serverEndpoint = new ServletContextHandler(ServletContextHandler.SESSIONS);
+		serverEndpoint.setContextPath("/");
+		server.setHandler(serverEndpoint);
+	        
+	        // Add a websocket to a specific path spec
+	        ServletHolder holderEvents = new ServletHolder("ws-events", WSServlet.class);
+	        serverEndpoint.addServlet(holderEvents, "/events/*");
+	       
+	        try
+	        {
+	            server.start();
+	            logger.info("Jetty server is up for websocket connections");
+	            
+	            //server.dump(System.err);
+	            //server.join();
+	            //logger.warn("server joined");
+	        }
+	        catch (Throwable t)
+	        {
+	        	logger.warn("server did not start");
+	            if(VERBOSE){t.printStackTrace(System.err);}
+	        }
+	}
+	
+
+	/**
+	 * Register for storage updates on the tables requested by web-gui
+	 * @param session		Session we are looking at
+	 * @param tableName		Tables that are associated with the session, to be registered
+	 * @return				Returns a HashMap that contains the sessions with their associated tables
+	 */
+	public HashMap<Session, List<String>> registerTable(Session session, String tableName){
+		
+		try {
+		List<String> tableNames = new ArrayList<String>();
+		storageSourceService.addListener(tableName, this);
+		logger.info("Added listener: "+ tableName);
+	
+		if(activeTables.containsKey(session)){
+			tableNames = activeTables.get(session);
+			tableNames.add(tableName);
+			activeTables.put(session, tableNames);
+		}else{
+			tableNames.add(tableName);
+			activeTables.put(session, tableNames);
+		}
+		
+		} catch (StorageException ex) {
+			logger.error("Error in installing listener for "
+					+ "switch table {}");
+		}
+		return activeTables;
+	}
+	
+
+	/**
+	* Sets the IStorageSource to use for Topology
+	*
+	* @param storageSource
+	*            the storage source to use
+	*/
+	public void setStorageSource(IStorageSourceService storageSourceService) {
+		this.storageSourceService = storageSourceService;
+	}	
+	
+	/**
+	 * Gets the storage source for this ITopology
+	 *
+	 * @return The IStorageSource ITopology is writing to
+	 */
+	public IStorageSourceService getStorageSource() {
+		return storageSourceService;
+	}
+	
+	@Override
+	public void rowsModified(String tableName, Set<Object> rowKeys) {
+			if(VERBOSE){logger.info("Table: " + tableName + " was modified");}
+			readTableFromStorage(tableName);
+	}
+
+	@Override
+	public void rowsDeleted(String tableName, Set<Object> rowKeys) {
+			if(VERBOSE){logger.info("Table: " + tableName + " had rows deleted");}
```

Comment:
Can you just get rid of these statements and make it logger.debug? its pretty much the same thing just it is controlled for everything and will clean up the code.